.github/workflows/publish-docs.yml

```diff
@@ -0,0 +1,24 @@
+name: Publish Docs
+
+on:
+  push:
+    branches: [ master ]
+    tags:
+      - "*"
+    paths:
+      - "docs/**"
```

Comment:
watching for changes to `mkdocs.yml` would be good too

Reply:
done